mods/ra2/rules/allied-structures.yaml

```diff
@@ -349,6 +349,20 @@ gaairc:
 	SpawnSurvivors:
 		DeathTypes: ExplosionDeath, BulletDeath
 		Actors: e1, e1
+	SpawnNeighboringActors:
+		ActorTypes: gapad
+		Locations: 1,0, 1,1, 2,1
```

Comment:
Why only 3?
This seems to line up, in some way I don't yet understand, with the footprint having 3 `=`s, not 4 (or even 6 `x`s).

Reply:
The main building is already one pad.